website/src/pages/guides/dev-onboarding-how-is-it-unique/index.module.scss

```diff
@@ -0,0 +1,13 @@
+@import '@styles/colors';
+
+.hero {
+    background-image: url('/guides/dev-onboarding-unique/hero-mobile.png');
```

Comment:
We seem to be missing the relevant assets

Reply:
Thank you. Looking into this, and the assets in `public` are listed in `website/gitignore`. So, although I updated the directory with these files, I didn't notice that they were missing from the commit. What is our process for adding these assets from `public`, as it was done in `guides/continuous-developer-onboarding`?

Reply:
Thank you! It looks like the assets in `public` are listed in `website/gitignore`. So, although I added these files in the directory, I didn't notice that they were missing from the commit. What is our process for adding these assets from `public`, as it was done in `guides/continuous-developer-onboarding`?

Reply:
Hi @zlonko — our Gatsby project [builds assets from the `static` directory](https://www.gatsbyjs.com/docs/how-to/images-and-media/static-folder/#adding-assets-outside-of-the-module-system), so this is where the assets should be added. You can take a look at the [PR that added the `continuous-developer-onboarding`](https://github.com/sourcegraph/about/pull/5207/files) for guidance/ context on where the assets should go!

Reply:
@katjuell Thank you for linking me this information, Katy! I added the assets to their own folder at `static/guides/dev-onboarding-unique` per the PR context.